src/ray/raylet/node_manager.cc

```diff
@@ -396,7 +396,7 @@ void NodeManager::ProcessClientMessage(
   case protocol::MessageType::RegisterClientRequest: {
     auto message = flatbuffers::GetRoot<protocol::RegisterClientRequest>(message_data);
     if (message->is_worker()) {
-      // Create a new worker from the registration request.
+      // Create a new worker from the registration request. 
```

Comment:
Is this change unintended? if so, revert it.

Reply:
Yup.